draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -17,6 +17,7 @@ export default {
           'import { GuidanceBlock } from "@kaizen/draft-guidance-block";',
       },
     },
+    backgrounds: { default: "Stone" },
```

Comment:
The changes here weren't necessary, but having the story-level parameters sit closer to their corresponding story is more consistent with other stories in this repo.